blueocean-config/src/main/java/io/jenkins/blueocean/config/BlueOceanConfig.java

```diff
@@ -0,0 +1,14 @@
+package io.jenkins.blueocean.config;
+
+import org.kohsuke.stapler.export.ExportedBean;
+
+/**
+ * Holds the Blue Ocean configuration
+ *
+ */
+@ExportedBean
+public abstract class BlueOceanConfig {
```

Comment:
Please move it to `io.jenkins.blueocean.rest.model` package in blueocean-rest module.